index.js

```diff
@@ -1,28 +1,65 @@
 const readline = require('readline');
 const readlineInterface = readline.createInterface(process.stdin, process.stdout);
 
+
+
+function outsideRoom (answer) {
+if (answer === 'read sign'){
+  return 'The sign says "Welcome to Burlington Code Academy! Come on up to the third floor. If the door is locked, use the code 12345."'
+}else if (answer === 'gargle'){
+  return "Sorry, I don't know how to gargle."
+}else if(answer === 'take sign'){
+  return 'That would be selfish. How will other students find their way?'
```

Comment:
Just returning these won't display the text to the terminal. Try `console.log` instead